test/TestAsset/reflectionTestFunction.php

```diff
@@ -23,3 +24,4 @@
 function reflectionTestFunction(bool $arg1, $arg2, string $arg3 = 'string', $arg4 = 'array')
 {
 }
+// phpcs:enable
```

Comment:
And here, too.

Reply:
In the current implementation where DocBlock param tags are parsed, it is not possible to satisfy CS. This has to be reviewed with a possible change of parsing argument type declarations.